### ISSUE TITLE
Generate the table from another file 

### ISSUE THREAD

**Issue by Ryu1845** [Vaibhavs10/open-tts-tracker#30]
Hi, currently every modification has to be done directly in markdown which is pretty unwieldy. So I propose to move to data to a separate file like a csv, yaml, or json. This would have the added benefit making the data easily machine-readable.

If you accept this proposal, I would probably implement this myself if all goes well. The generation part should be trivial with a github action.

**Comment by Pendrokar** [comment]
Meanwhile use this as a workaround:
https://www.tablesgenerator.com/markdown_tables#

**Comment by Vaibhavs10** [comment]
Agree, maybe we can use a Hugging Face datasets? Does that work? https://huggingface.co/datasets

We can have a dataset and then use that as the ground truth?

**Comment by Pendrokar** [comment]
> Agree, maybe we can use a Hugging Face datasets? Does that work? https://huggingface.co/datasets
> 
> We can have a dataset and then use that as the ground truth?
@Vaibhavs10 

Here is a start with just the capabilities table and the TTS info within README.md:
https://huggingface.co/datasets/Pendrokar/open_tts_tracker

But I used a CSV/TSV so that Dataset Viewer would auto-convert it.

It becomes less unwieldy to modify when using a Rainbow CSV extension for your IDE.
